app/assets/javascripts/ref_status_typeahead.js

```diff
@@ -73,7 +73,7 @@ function refStatusTypeahead(options){
       var item = $("<li>");
       $ref_problem_list.append(item);
       // State and status comes from GitHub or Samson
-      item.html(status.state + ": " + status.description);
+      item.html(status.state + ": " + (status.description || status.context));
```

Comment:
```suggestion
      item.html(status.state + ": " + (status.description.presence || status.context));
```

Reply:
also should probably be using `item.text()` to avoid XSS

Reply:
we have lots of users that need html:
```
Deploy.first(100).uniq { |d| d.project_id }.map { |d| CommitStatus.new(d.project, d.reference, stage: d.stage).statuses }
```
so added sanitization in the controller

Reply:
<img width="598" alt="Screen Shot 2021-03-04 at 10 41 44 AM" src="https://user-images.githubusercontent.com/11367/110013228-3f31b580-7cd6-11eb-915d-613b6cdf138b.png">